src/pages/CertificatesViewAll.js

```diff
@@ -60,25 +109,11 @@ export default function CertificatesViewAll() {
         )}
         {data && (
           <Table
-            action={(item) =>
-              navigate(`/certificate/${item.original_token_id || item.id}`)
+            action={(cert) =>
```

Comment:
Just finished adding a cert from Heidi's point of view ( did not finish adding Co2 ) . When I click on this I get redirected to `/certificate/undefined` .. any idea ? ( also , why can't i see the hand cursor when hovering it , is it just me )?

![Screen Recording 2024-03-07 at 09 35 39](https://github.com/digicatapult/sqnc-hyproof-client/assets/58742260/3740367c-bd25-4f71-bb0e-8a0dddae4a63)

Reply:
Hand cursor can't be added here. This should be backlogged for UI component library where table lives to update cursor

Reply:
On 

```
hen I click on this I get redirected to /certificate/undefined .. any idea ? ( also , why can't i see the hand cursor when hovering it , is it just me )?
```

I will have to come back.

Reply:
Ok, this looks like the sync up did not happen across your docker.

The correct behaviour with cursor and other elements like highlight should be backlogged as designs did not have either 

CC @mattdean-digicatapult @elisehaldane

Reply:
why are you including Elle @n3op2 she's not working on this? Designs aren't complete and I believe there is a ticket in the backlog to remove hover behaviour on the timeline (it was in Rob's snag list)

Reply:
As far as I can tell , `cert` is an array . Ignoring the cursor, the point is it is not working  .. also my api is synchronised ..

Reply:
Nice, this one is sorted and pushed was already clear after the catch up